src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -39,8 +40,31 @@ export function isCodeTargetProperties(props: TargetProperties): props is CodeTa
     return props.target === CODE_TARGET_TYPE
 }
 
+/**
+ * Creates a description for a SAM debugconfig entry (in launch.json).
+ *
+ * Example: `makeName('foo', '/bar/baz', 'zub')` => `"baz:foo (zub)"
+ *
+ * @param primaryName
+ * @param parentDir  Optional directory name (used as a prefix)
+ * @param suffix  Optional info used to differentiate the name
+ */
+function makeName(primaryName: string, parentDir: string | undefined, suffix: string | undefined) {
```

Comment:
Consider destructuring for these. Easy to get the parameter order incorrect

Reply:
it's a private/local function covered by tests